test/functional/test_framework/authproxy.py

```diff
@@ -191,3 +191,6 @@ def _get_response(self):
         else:
             log.debug("<-- [%.6f] %s"%(elapsed,responsedata))
         return response
+
+    def __truediv__(self, relative_uri):
+        return AuthServiceProxy("{}{}".format(self.__service_url, relative_uri), self._service_name, connection=self.__conn)
```

Comment:
In commit "[QA] Fix AuthProxy multiwallet support"

Should drop this commit. This should be {}/{} for consistency with python's pathlib API, and you should change all the / "/path" expressions to / "path" again for consistency with normal python path expressions. (Personally also I think the double slashes look strange).